composer/utils/run_directory.py

```diff
@@ -1,5 +1,42 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""The run directory stores artifacts created during training.
+
+Callbacks, loggers, the checkpointer, and the profiler can write files to and read files from the run
+directory. For example,
+
+* The :class:`~composer.trainer.checkpoint.CheckpointSaver` can save checkpoints inside the run directory.
+* The :class:`~composer.loggers.file_logger.FileLogger` can save log files inside the run directory.
+* The :class:`~composer.profiler.profiler.Profiler` can save profiling traces inside the run directory.
```

Comment:
I assume the path should be `~composer.core.profiler.profiler`, and not one of the profilers in`~composer.profiler` (e.g. `dataloader_profiler`, `system_profiler`)

Reply:
Yup, I wrote this in anticipation of Bandish's PR. Need to validate that the link is correct...

Reply:
Fixed